<launch>
  <arg name="host" default="localhost"/>
  <arg name="port" default="2000"/>
  <arg name="fixed_delta_seconds" default="0.05"/>
  <arg name="no_rendering_mode" default="true"/>
  <arg name="synchronous_mode" default="true"/>

  <group ns="carla">
    <node pkg="conformal_lattice_planner"
      type="random_traffic_node"
      name="carla_simulator"
      output="log"
      required="true">
      <!-- connect to client -->
      <param name="host" value="$(arg host)"/>
      <param name="port" value="$(arg port)"/>
      <!-- world settings -->
      <param name="fixed_delta_seconds" value="$(arg fixed_delta_seconds)"/>
      <param name="no_rendering_mode" value="$(arg no_rendering_mode)"/>
      <param name="synchronous_mode" value="$(arg synchronous_mode)"/>
    </node>
  </group>
</launch>
